x-pack/plugins/dashboard_enhanced/public/services/drilldowns/dashboard_to_dashboard_drilldown/drilldown.tsx

```diff
@@ -89,7 +90,7 @@ export class DashboardToDashboardDrilldown
     };
 
     if (context.embeddable) {
-      const input = context.embeddable.getInput();
+      const input = context.embeddable.getInput() as Readonly<SearchInput>;
```

Comment:
@ThomThomson why `Readonly<SearchInput>` ? shouldn't query, filters and range be on embeddableinput directly ?
cc @streamich

Reply:
They will be, I'll merge that PR that adds query, filters and range onto base embeddable.